bokun.yaml

```diff
@@ -5537,413 +5734,535 @@ definitions:
+        items:
+          $ref: '#/definitions/CustomerFieldReferenceDto'
+      inclusions:
+        type: array
+        items:
+          type: string
           enum:
-          - STROLLER_OR_PRAM_ACCESSIBLE
-          - WHEELCHAIR_ACCESSIBLE
-          - LIMITED_MOBILITY_ACCESSIBLE
-          - LIMITED_SIGHT_ACCESSIBLE
-          - ANIMALS_OR_PETS_ALLOWED
-          - PUBLIC_TRANSPORTATION_NEARBY
-          - INFANT_SEATS_AVAILABLE
-          - INFANTS_MUST_SIT_ON_LAPS
-          - PASSPORT_REQUIRED
-          - DRESS_CODE
+            - ENTRY_OR_ADMISSION_FEE
+            - FUEL_SURCHARGE
+            - LANDING_AND_FACILITY_FEES
+            - PARKING_FEES
+            - ENTRY_TAX
+            - DEPARTURE_TAX
+            - NATIONAL_PARK_ENTRANCE_FEE
+            - GOODS_AND_SERVICES_TAX
+            - TIP_OR_GRATUITY
+            - FOOD_AND_DRINKS
+            - WIFI
+            - BUS_FARE
+      included:
+        type: string
+        description: 'Optional text describing what is included when you book this product. May contain HTML markup.'
+      exclusions:
+        type: array
+        items:
           type: string
+          enum:
+            - ENTRY_OR_ADMISSION_FEE
+            - FUEL_SURCHARGE
+            - LANDING_AND_FACILITY_FEES
+            - PARKING_FEES
+            - ENTRY_TAX
+            - DEPARTURE_TAX
+            - NATIONAL_PARK_ENTRANCE_FEE
+            - GOODS_AND_SERVICES_TAX
+            - TIP_OR_GRATUITY
+            - FOOD_AND_DRINKS
+            - WIFI
+            - BUS_FARE
+      excluded:
+        type: string
+        description: 'Optional text describing what is excluded when you book this product. May contain HTML markup.'
+      requirements:
+        type: string
+        description: 'The text describing what you need to bring for this activity. May contain HTML markup.'
+      knowBeforeYouGoItems:
         type: array
+        items:
+          type: string
+          enum:
+            - STROLLER_OR_PRAM_ACCESSIBLE
+            - WHEELCHAIR_ACCESSIBLE
+            - LIMITED_MOBILITY_ACCESSIBLE
+            - LIMITED_SIGHT_ACCESSIBLE
+            - ANIMALS_OR_PETS_ALLOWED
+            - PUBLIC_TRANSPORTATION_NEARBY
+            - INFANT_SEATS_AVAILABLE
+            - INFANTS_MUST_SIT_ON_LAPS
+            - PASSPORT_REQUIRED
+            - DRESS_CODE
       attention:
         type: string
         description: 'Text describing any points that need special attention for the passengers. May contain HTML.'
-      baseLanguage:
-        type: string
-        description: 'The language that the product description is written in. There may also be translations available.'
-      bookableExtras:
+      locationCode:
+        $ref: '#/definitions/LocationCodeDto'
+        description: 'The city that represents the main location for this activity. Specified as a location from the United Nations Code for Trade and Transport Locations (UN/LOCODE).'
```

Comment:
We should mark this as deprecated. You can no longer set this in the UI.

Reply:
Done